spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -1468,6 +1468,11 @@
       "type": "java.lang.Boolean",
       "description": "Whether pooling is enabled. Enabled automatically if \"r2dbc-pool\" is on the classpath."
     },
+    {
+      "name": "spring.r2dbc.proxy.enabled",
+      "type": "java.lang.Boolean",
+      "description": "Whether to proxy ConnectionFactory. Enabled automatically if \"r2dbc-proxy\" is on the classpath."
```

Comment:
That description does not seem accurate. "Whether to proxy the ConnectionFactory if necessary"  with a `defaultValue: true` sounds a better option to me.